docs/source/api/apollo-server.mdx

```diff
@@ -668,8 +515,7 @@ const result = await server.executeOperation({
 The `executeOperation` method takes two arguments:
 * The first argument is an object describing the GraphQL operation to be executed.
   * Supported fields are listed in the table below.
-* The second optional argument is passed to `ApolloServer`'s `context` function.
-  * This is helpful when [testing](../testing/testing/) that your server's `context` is correctly gathering the values needed from a request.
+* The second optional argument is used as the operation's [context value](../data/resolvers/#the-context-argument).
```

Comment:
Note that it's "optional" in the sense that you should pass it if your server expects a context value, and you should only leave it out if your server only expects the default empty `{}` as context value.

Reply:
How does this feel for explaining that:

> The second optional argument is used as the operation's [context value](../data/resolvers/#the-context-argument). Note, this argument is only optional if your server _doesn't_ expect a context value (i.e., your server uses the default context because you didn't explicitly provide another one).